src/util/updateSitemap.ts

```diff
@@ -36,7 +38,9 @@ export type Element = {
 const writeFile = promisify(fs.writeFile);
 
 const LANGUAGES = ['en', 'fi', 'sv'];
-const STATIC_URLS = ['home'];
+const STATIC_URLS = ['home', 'events', 'collections'].concat(
+  Object.keys(MAPPED_PLACES)
+);
```

Comment:
To me `const STATIC_URLS = ['home', 'events', 'collections', ...Object.keys(MAPPED_PLACES)];` would be easier to read. But it's really just matter of taste

Reply:
.concat came first to my mind because I saw it used somewhere recently, I agree that spread is better here